das-cli/src/commands/vault/vault_cli.py

```
@@ -0,0 +1,266 @@
+        self,
+        settings: Settings,
+        vault_container_manager: VaultContainerManager,
+    ) -> None:
+        super().__init__()
+        self._settings = settings
+        self._vault_container_manager = vault_container_manager
+
+    def _get_container(self):
+        return self._vault_container_manager.get_container()
+
+    def run(self, prune: bool = False):
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)